### ISSUE TITLE
[Package Issue]: ElectronicArts.EADesktop version 12.143.0.5400 Installer hash does not match

### ISSUE THREAD

**Issue by HulkSmashBurgers** [microsoft/winget-pkgs#101564]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installer hash mismatch.

### Brief description of your issue

when attempting to install ElectronicArts.EADesktop, installation fails with "Installer hash does not match; this cannot be overridden when running as admin"

### Steps to reproduce

1. winget install ElectronicArts.EADesktop
2. wait for the download to complete
3. Winget will report "Installer hash does not match; this cannot be overridden when running as admin"

### Actual behavior

Winget reports "Installer hash does not match; this cannot be overridden when running as admin"

### Expected behavior

Winget installs ElectronicArts.EADesktop successfully.

### Environment

```raw
Windows Package Manager v1.4.10173
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22621.1413
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.19.10173.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

User Settings: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\settings.json
```

### Screenshots and Logs

[WinGet-2023-04-04-22-47-11.713.log](https://github.com/microsoft/winget-pkgs/files/11155088/WinGet-2023-04-04-22-47-11.713.log)
![Screenshot 2023-04-04 230733](https://user-images.githubusercontent.com/2564116/229978673-f1f921c1-7502-4261-8372-1a1bafa0237a.png)

**Comment by BrandonWanHuanSheng** [comment]
It look like need Newer Version Value. Please give me. Otherwise I unable to run wingetcreate  because newer than winget

**Comment by BrandonWanHuanSheng** [comment]
The issue might take effect in the early morning.

**Comment by BrandonWanHuanSheng** [comment]
Finally, it appeared.

**Comment by mdanish-kh** [comment]
Close with reason: Resolved in #101762;
